change/@uifabric-react-cards-2020-08-20-11-57-01-cardImports.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Maybe make this minor? (I'm not sure what the standard way to handle this case is)
```suggestion
  "type": "minor",
```

---

I think it shouldn't be minor since we are not changing the public facing API?

---

I sometimes also use minor for potentially-more-impactful changes even if they don't change the API. Up to you.